auth.go

```diff
@@ -308,9 +308,18 @@ func (a *Auth) SignOut(ctx context.Context, userToken string) error {
 	return nil
 }
 
-// InviteUserByEmail sends an invite link to the given email. Returns a user.
-func (a *Auth) InviteUserByEmail(ctx context.Context, email string) (*User, error) {
-	reqBody, _ := json.Marshal(map[string]string{"email": email})
+// InviteUserByEmailWithOpts sends an invite link to the given email with metadata. Returns a user.
+func (a *Auth) InviteUserByEmailWithData(ctx context.Context, email string, data map[string]interface{}, redirectTo string) (*User, error) {
+	params := map[string]interface{}{"email": email}
+	if data != nil {
+		params["data"] = data
+	}
+
+	if redirectTo != "" {
+		params["redirectTo"] = redirectTo
```

Comment:
Should ```params["redirectTo"]``` be ```params["redirect_to"]``` instead?

See https://github.com/supabase/auth-js/blob/92fefbd49f25e20793ca74d5b83142a1bb805a18/src/lib/fetch.ts#L147